### PR TITLE
[flutter_image][ci] ignore deprecation warning, migrate master references to main.

### PR DESCRIPTION
**Pull request by cyanglaz**
See: https://github.com/flutter/flutter/issues/105336, this PR is the first half to fix the issue.

This PR also:
~~1. sets the git FETCH_HEAD to main to fix ci, because migrated the repot to main.~~ (fixed by https://github.com/flutter/packages/pull/2200#event-6740294133)
2. migrate other references to the master branch to main.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran the auto-formatter. (Unlike the flutter/flutter repo, the flutter/packages repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the package surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [x] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/packages/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/packages/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

**Comment by cyanglaz** [comment]
We need to land the fix for git first to make the CI happy.

File: packages/flutter_image/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 4.1.3
+
+* Suppress deprecation warnings.
```
Comment (ditman) [inline packages/flutter_image/CHANGELOG.md:3]:
```suggestion
* Suppresses deprecation warnings.
```

File: packages/extension_google_sign_in_as_googleapis_auth/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 2.0.5
+
+* Update README to reference the correct github URL.
```
Comment (ditman) [inline packages/extension_google_sign_in_as_googleapis_auth/CHANGELOG.md:3]:
```suggestion
* Updates README to reference the correct github URL.
```

File: packages/pointer_interceptor/CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,8 @@
-## NEXT
+## 0.9.3+2
 
 * (Temporarily) helps tests introduced in prior version to pass in `stable`.
   (This will be removed when `master` rolls to `stable`)
+* Update README to reference the correct github URL.
```
Comment (ditman) [inline packages/pointer_interceptor/CHANGELOG.md:5]:
```suggestion
* Updates README to reference the correct github URL.
```

File: packages/rfw/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
+## 1.0.5
+
+* Fix URL in document.
```
Comment (ditman) [inline packages/rfw/CHANGELOG.md:3]:
```suggestion
* Fixes URL in document.
```

File: third_party/packages/cupertino_icons/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,6 @@
+## 1.0.5
+* Update README to reference correct URL.
+
 ## 1.0.4
 * Update README to link to API docs.
```
Comment (ditman) [inline third_party/packages/cupertino_icons/CHANGELOG.md:6]:
```suggestion
## 1.0.5

* Updates README to reference correct URL.

## 1.0.4

* Updates README to link to API docs.

```

**Review by ditman — APPROVED** [review]
Tweaked changelog entries to use "present tense indicative", and conform with the style guide. [Link](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style).

**Comment by ditman** [comment]
This should be enough for the roller to succeed. Thanks for cleaning up the rest of the `master` entries!

**Pull request merged**